app/models/manageiq/providers/openstack/inventory/parser.rb

```diff
@@ -38,4 +38,10 @@ def orchestration_template(stack)
       o
     end
   end
+
+  private
+
+  def compose_ems_ref(*keys)
+    keys.join('_')
+  end
```

Comment:
This was the only method left being used from the `ManageIQ::Providers::Openstack::RefreshParserCommon::OrchestrationStacks` module allowing us to delete it